### PR TITLE
Hotfix/1.0.1

### PR DESCRIPTION
**Pull request by KarmaKamikaze**

### CODE REVIEW

**Review by mirakst — DISMISSED** [review]
OkyesGIF

File: README.md [hunk 1]
```diff
@@ -1,9 +1,57 @@
...
+| setup           | Contains code that should be run at initialization of program                       |
+| loop            | Contains code that will run repeatedly until Arduino device is turned off           |
+| functions       | Contains function definitions [^function]                                           |
+
+[^global]: A global variable is accessible from anywhere in the program
+[^function]: A procedure that can be executed from anywhere in the program
+
+3. Understand the keywords, statements and control structures
+
+STEP is formally documented in the [development journal](https://www.youtube.com/watch?v=dQw4w9WgXcQ).
```
Comment (damniko) [inline README.md:46]: ☣

**Review by damniko — COMMENTED** [review]
WAUW

**Review by damniko — DISMISSED** [review]
WAUW 2: now approved

**Review by mirakst — DISMISSED** [review]
Changes by @damniko:
ThumbsDownGladiatorGIF
Everything else looks great

**Comment by WassawRoki** [comment]
image

**Review by damniko — APPROVED** [review]

**Review by WassawRoki — APPROVED** [review]
image

**Review by mirakst — APPROVED** [review]
WooRageGIF

**Comment by Sarmisuper** [comment]
Im gay
lgtm.Final.final

**Comment by KarmaKamikaze** [comment]
[Hotfix linux processes](https://trello.com/c/f5J65Avk/134-hotfix-linux-processes)

**Pull request merged**